Validation/SiTrackerPhase2V/python/Phase2ITValidateRecHit_cfi.py

```diff
@@ -0,0 +1,128 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
+rechitValidIT = DQMEDAnalyzer('Phase2ITValidateRecHit',
+                            Verbosity = cms.bool(False),
+                            TopFolderName = cms.string("TrackerPhase2ITRecHitV"),
+                            ITPlotFillingFlag = cms.bool(False),
+                            rechitsSrc = cms.InputTag("siPixelRecHits"),
+                            InnerPixelDigiSource   = cms.InputTag("simSiPixelDigis","Pixel"),                          
```

Comment:
@sroychow in the premixing case this input needs to be customized.
See please how it is done here:

https://github.com/cms-sw/cmssw/blob/9e912a119e2bf295939ef033dbc5fb485df778c9/Validation/SiTrackerPhase2V/python/Phase2TrackerValidateDigi_cfi.py#L129-L135